app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/language_switcher.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import cats.syntax.eq._
+@import cats.instances.string._
+@import play.api.i18n.Messages
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.routes
+
+@this()
+
+@(enableLanguageSwitching: Boolean)(implicit messages: Messages)
+
+@if(enableLanguageSwitching) {
+<nav aria-label="@messages("language-switcher.nav")" class="language-switcher pull-right">
+<ul>
+    <li>
+        @if(messages.lang.code === "en") {
+        <span>@messages("language-switcher.english")</span>
+        <span aria-hidden="true"> | </span>
+        <a id="switch-language" href="@routes.CustomLanguageController.switchToWelsh" data-journey-click="link - click:Language selector:Welsh">
+            <span class="visuallyhidden">@messages("language-switcher.welsh.hidden")</span>
+            <span aria-hidden="true">@messages("language-switcher.welsh")</span>
+        </a>
+        } else {
+        <a id="switch-language" href="@routes.CustomLanguageController.switchToEnglish" data-journey-click="link - click:Language selector:English">
+            <span class="visuallyhidden">@messages("language-switcher.english.hidden")</span>
+            <span aria-hidden="true">@messages("language-switcher.english")</span></a>
+        <span aria-hidden="true"> | </span>
+        <span>@messages("language-switcher.welsh")</span>
+        }
+    </li>
+</ul>
+</nav>
+}
```

Comment:
@nubz added this in just to test - change as required

Reply:
LGTM